src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTestBase.cs

```diff
@@ -51,6 +51,12 @@ protected static void VerifyModelForDeclarationPattern(SemanticModel model, Sing
             VerifyModelForDeclarationPattern(model, decl, false, references);
         }
 
+        protected static void VerifyModelForDeclarationPatternWithoutDataFlow(SemanticModel model, SingleVariableDesignationSyntax decl, params IdentifierNameSyntax[] references)
+        {
+            // TODO: should anything be different here?
```

Comment:
>// TODO: [](start = 12, length = 8)

I believe our current policy is to not merge with TODO comments into a branch that is not a feature branch.